src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -223,7 +223,7 @@ public void initializeFromEntity(@NotNull EntityCitizen entity)
         maxHealth = entity.getMaxHealth();
         experience = 0;
         level = 0;
-        @NotNull Random random = new Random();
+        final @NotNull Random random = new Random();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)